noxfile.py

```diff
@@ -250,16 +259,16 @@ def docker_qa_build_compiler(session: nox.Session, version: str) -> None:
 @nox.session(python=False)
 def docker_qa_run(session: nox.Session) -> None:
     """Run the docker container for the default GCC version."""
-    session_id = f"docker_qa_run_compiler({GCC_VERSION2USE})"
+    session_id = f"docker_qa_run_({GCC_VERSION2USE})"
     session.log(f"Notify session {session_id}")
     session.notify(session_id)
 
 
-@nox.session(python=False)
-def docker_qa_run_all_compiler(session: nox.Session) -> None:
+@nox.session(python=False, name="docker_qa_run_compiler(all)")
+def docker_qa_run_compiler_all(session: nox.Session) -> None:
     """Run the docker container for the all GCC versions."""
     for version in GCC_VERSIONS:
-        session_id = f"docker_qa_run_compiler({version})"
+        session_id = f"docker_qa_run_({version})"
```

Comment:
Old naming convention?

```suggestion
        session_id = f"docker_qa_run_compiler({version})"
```